lib/msal-browser/docs/caching.md

```diff
@@ -71,6 +71,9 @@ To faciliate efficient token acquisition while maintaining a good UX, MSAL cache
 > :bulb: The authorization code is only stored in memory and will be discarded after redeeming it for tokens.
 
 ## Warning :warning:
+
+**NOTE: `temporaryCacheLocation` is deprecated as of MSAL v5 and will be removed in a future release.**
+
 Overriding `temporaryCacheLocation` should be done with caution. Specifically when choosing `localStorage`. Interaction in more than one tab/window will not be supported and you may receive `interaction_in_progress` errors unexpectedly. This is an escape hatch, not a fully supported feature.
```

Comment:
We are keeping the docs? I assumed if it is removed, wouldn't it be just removal notice in migration guidance since this branch is V5?

Reply:
Thanks! This is now removed as it was deprecated for v4 and removed in v5.